react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -0,0 +1,252 @@
+         */
+        _peopleSearchUrl: React.PropTypes.string,
+
+        /**
+         * Invoked to obtain translated strings.
+         */
+        t: React.PropTypes.func
+    };
+
+    /**
+     * Initializes a new {@code AddPeopleDialog} instance.
+     *
+     * @param {Object} props - The read-only properties with which the new
+     * instance is to be initialized.
+     */
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            /**
+             * Indicating that an error occurred when adding people to the call.
+             */
+            addToCallError: false,
+
+            /**
+             * Indicating that we're currently adding the new people to the
+             * call.
+             */
+            addToCallInProgress: false,
+
+            /**
+             * The list of invite items.
+             */
+            inviteItems: new Immutable.List()
+        };
+
+        this._multiselect = null;
+        this._resourceClient = {
+            makeQuery: text => searchPeople(
+                this.props._peopleSearchUrl, this.props._jwt, text),
+            parseResults: response => response.map(user => {
+                const avatar = ( // eslint-disable-line no-extra-parens
+                    <Avatar
+                        size = 'medium'
+                        src = { user.avatar } />
+                );
+
+                return {
+                    content: user.name,
+                    value: user.id,
+                    elemBefore: avatar,
+                    item: user
+                };
+            })
+        };
+
+        this._isAddDisabled = this._isAddDisabled.bind(this);
+        this._onSelectionChange = this._onSelectionChange.bind(this);
+        this._onSubmit = this._onSubmit.bind(this);
+        this._setMultiSelectElement = this._setMultiSelectElement.bind(this);
+    }
+
+    /**
+     * React Component method that executes once component is updated.
+     *
+     * @param {Object} prevState - The state object before the update.
+     * @returns {void}
+     */
+    componentDidUpdate(prevState) {
+        /**
+         * Clears selected items from the multi select component on successful
+         * invite.
+         */
+        if (prevState.addToCallError
+            && !this.state.addToCallInProgress
+            && !this.state.addToCallError
+            && this._multiselect) {
+            this._multiselect.clear();
+        }
+    }
+
+    /**
+     * Renders the content of this component.
+     *
+     * @returns {XML}
+     */
+    render() {
+        return (
+            <Dialog
+                okDisabled = { this._isAddDisabled() }
+                okTitleKey = 'addPeople.add'
+                onSubmit = { this._onSubmit }
+                titleKey = 'addPeople.title'
+                width = 'small'>
+                { this._getUserInputForm() }
+            </Dialog>
+        );
```

Comment:
Pressing enter is still refreshing the page specifically when I have the "we stumbled a bit" message open. I can trigger the message by searching for curly braces.